pkg/migrations/seq_num.go

```diff
@@ -10,14 +10,14 @@ import (
 // It is a shared constant between central and the migrator binary.
 func CurrentDBVersionSeqNum() int {
 	if env.PostgresDatastoreEnabled.BooleanSetting() {
-		return internal.CurrentDBVersionSeqNum + internal.PostgresDBVersionPlus
+		return internal.CurrentDBVersionSeqNum
 	}
-	return internal.CurrentDBVersionSeqNum
+	return internal.LastRocksDBVersionSeqNum
 }
 
-// CurrentDBVersionSeqNumWithoutPostgres is the base of current DB version number
-// without Postgres migrations. This function should only be used in Postgres
+// BasePostgresDBVersionSeqNum is the base of DB version number
+// for Postgres migrations. This function should only be used in Postgres
 // migrations.
-func CurrentDBVersionSeqNumWithoutPostgres() int {
-	return internal.CurrentDBVersionSeqNum - 1
+func BasePostgresDBVersionSeqNum() int {
```

Comment:
What is the lifetime of this function ? Should it be restrained to the initial Postgres DB load, or should it be used in newer migrations too, or could it be dropped so `CurrentDBVersionSeqNum` can be used directly in all n_ migrations ?

Reply:
This should not be used in new Postgres to Postgres migrations. In your PR, I can make change accordingly after your PR merged.